pkg/build/python.go

```diff
@@ -34,11 +37,13 @@ func python(root string, args Args) (string, error) {
 
 	var buf strings.Builder
 	if err := t.Execute(&buf, struct {
-		Base       string
-		Entrypoint string
+		Base         string
+		Entrypoint   string
+		Requirements bool
```

Comment:
make it clearer it's a bool?

```suggestion
		RequirementsExist bool
```

Reply:
Renamed to `HasRequirements`.